x/concentrated-liquidity/lp_test.go

```diff
@@ -854,15 +862,17 @@ func (s *KeeperTestSuite) TestUpdatePosition() {
 			numPositions:   1,
 			expectedError:  true,
 		},
-		"try updating with ticks outside existing position's tick range - error because fee accumulator is uninitialized": {
-			poolId:         1,
-			ownerIndex:     0,
-			lowerTick:      DefaultUpperTick + 1,
-			upperTick:      DefaultUpperTick + 100,
-			joinTime:       DefaultJoinTime,
-			liquidityDelta: DefaultLiquidityAmt,
-			numPositions:   1,
-			expectedError:  true,
+		"try updating with ticks outside existing position's tick range - initializes the accummulator": {
```

Comment:
```suggestion
		"try updating with ticks outside existing position's tick range - initializes the accumulator": {
```

Reply:
Also, "try" implies this test case should error

Reply:
Did a double take on this test case – I would have expected it to error since the ticks don't line up, but since `positionId` is not specified (and therefore set to 0), it ends up initializing a new position at ID 0. Is this the intended behavior @p0mvn?

Reply:
I added a new method that does proper validation of the attempted update:
[106f64e](https://github.com/osmosis-labs/osmosis/pull/4973/commits/106f64ee4fd729bac8b338cf5d10e142d430b2a1)

Updated the test cases per requests